redaxo/src/addons/mediapool/lib/mediapool.php

```diff
@@ -122,7 +122,10 @@ public static function isAllowedExtension(string $filename, array $args = []): b
         foreach ($blockedExtensions as $blockedExtension) {
             // $blockedExtensions extensions are not allowed within filenames, to prevent double extension vulnerabilities:
             // -> some webspaces execute files named file.php.txt as php
-            if (str_contains($filename, '.' . $blockedExtension)) {
+            if ($fileExt !== '' &&
+                str_ends_with($filename, '.' . $blockedExtension) && // Prüfe ob der String mit der verbotenen Endung endet
+                !str_ends_with($filename, '.' .  $blockedExtension . '.' . $fileExt) //prüfe ob es keine doppelte Endung der Form *.php.ext gibt
+            )
```

Comment:
Syntaxfehler, hier fehlt die geschweifte Klammer:

```suggestion
            ) {
```